getTest.yml

```diff
@@ -0,0 +1,15 @@
+config:
+  target: 'http://localhost:3003'
+  phases:
+    - duration: 60
+      arrivalRate: 10
+scenarios:
+  - flow:
+    - get:
```

Comment:
You should randomize these endpoints if you have the time since the data gets cached after repeated visits

Reply:
Fixed